utilities/_layout.scss

```diff
@@ -0,0 +1,32 @@
+//Full responsive layout based on design system
+//Todo: amend spacing calc for between 414px and 599 as some misalignment
+.grid {
+    @include grid(repeat(6, 1fr), 16px, 24px);
+    @media ($mobile-upper <= width <= $desktop-lower) {
+        @include grid(repeat(12, 1fr), 32px, 75px);
+    }
+    @media (width >= ($desktop-lower + 1)) {
+        @include grid(repeat(12, 1fr), 32px, 100px);
+    }
+}
+
+.full-width {
+    grid-column: span 6;
+    @media (width >= $mobile-upper) {
+        grid-column: span 12;
+    }
+}
+
+.col-3 {
+    grid-column: span 6;
+    @media (width >= $mobile-upper) {
+        grid-column: span 4;
```

Comment:
Should this be span 3?

Reply:
col-3 is more a third so at desktop takes up 4 columns so yes good shout it is confusing naming. 

I think one-third, one-quarter, half-width, full width work for 4 column, 3 column, 6 column and 12 column basis and at mobile they are all stacked from what I see in system

Reply:
Ah I see. I thought it was an error. I thought it referred to the same naming convention Bootstrap used on theirs; where they use col-span-1 to col-span 12, on a 12 column grid.

Yeah or using `.col-1-3`  or `.col-1/3` but using the slash doesn't feel right in CSS, i'm sure there will be a browser which wont support it.

Reply:
Just the footer in mobile has two column links - But I have done this specific styling within the Footer component itself.

Reply:
Have changed the naming convention for the columns to be more legibleand present in the component library